python/rikai/spark/functions/vision.py

```diff
@@ -158,15 +160,18 @@ def video_to_images(
     return [
         Image.from_array(
             img,
-            "{}_{}.jpg".format(base_path, (start_frame + idx) * sample_rate),
+            os.path.join(
```

Comment:
nit, use `pathlib.Path`?

Reply:
it doesn't work for non-local paths (e.g., it removes one of the slashs from https:// or s3://)

Reply:
👌